content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -4,7 +4,12 @@ linkTitle: Configuration
 weight: 10
 spelling:
   cSpell:ignore akka autoconfiguration Autoconfiguration Dotel HSET javaagent
-  cSpell:ignore serverlessapis Servlet servlet
+  cSpell:ignore serverlessapis servlet Customizer classloaders logback
```

Comment:
I notice start-cap on `Customizer` but I suppose case is probably not important.

Reply:
it's like "if upper case, the case counts, if lowercase it's both upper & lower case", so yes we are good here :)